src/App.tsx

```diff
@@ -29,10 +29,10 @@ function Clock({
     <div className="p-8">
       <div className="aspect-square border-black border-4 rounded-full grid place-items-center text-3xl relative">
         <span className="font-mono" ref={timeContainer}></span>
-        <img
+        {twitter && <img
```

Comment:
does it render fine when no twitter set? could try to find a different unavatar.io source for tris (is she on gravatar?) or allow passing an image url (e.g. to hotlink mastodon pfp)

Reply:
ngl afaik none of these are rendering correctly through twitter, it may have mostly broken since the whole X stuff, i'll take a look at alternatives

Reply:
is unavatar just dead? insta provider seems to be borked too. maybe we could use the mastodon API to grab someone's profile pic URL from their handle?

Reply:
honestly github might be an easy way to do it as well?

Reply:
closing, we decided this is out of scope of this MR